kvdb/go.mod

```diff
@@ -113,4 +137,6 @@ replace github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.11
 // https://deps.dev/advisory/OSV/GO-2021-0053?from=%2Fgo%2Fgithub.com%252Fgogo%252Fprotobuf%2Fv1.3.1
 replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 
-go 1.19
+replace github.com/lightningnetwork/lnd/sqldb => ../sqldb
```

Comment:
Same here re removing the replace.

Reply:
I think this is also needed as the PR modifies `sqldb` and is used in `kvdb`.

Reply:
Added a comment to clarify.